package.json

```diff
@@ -50,5 +50,10 @@
       "tokenRef": "GITHUB_AUTH"
     },
     "npm": false
+  },
+  "pnpm": {
+    "overrides": {
+      "ember-source": "4.12.2"
+    }
```

Comment:
Do we still need this?

Reply:
Good catch, I don't think so, will remove that ✅

Reply:
Thanks 🙏

Reply:
Removing this leads to failures on the embroider jobs 😞 
https://github.com/qonto/ember-autofocus-modifier/actions/runs/6494674434/job/17638109873?pr=447